### PR TITLE
[ServiceNow] add connector UI test

### PR DESCRIPTION
**Pull request by mmelko**
//test: @service-now

### CODE REVIEW

File: utilities/src/main/java/io/syndesis/qe/servicenow/ServiceNow.java [hunk 1]
```diff
@@ -144,4 +144,29 @@ public static void deleteIncident(String sysId) {
...
+
+    public static String getIncidentJson(Incident i) {
+        try {
+            return om.writeValueAsString(i);
+        } catch (JsonProcessingException e) {
+            fail("Unable to convert incident to json", e);
+        }
+        return null;
+    }
+>>>>>>> cea0dcc... [ServiceNow] add connector UI test
```
Comment (avano) [inline utilities/src/main/java/io/syndesis/qe/servicenow/ServiceNow.java]: this probably should not be here

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by tplevko** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mmelko** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Pull request merged**